### DISCUSSION TITLE
Consider setting explicitly the Chromium version

### DISCUSSION THREAD

**Discussion by giamma** [gotenberg/gotenberg#948] [Q&A] [answered]
If I am not mistaken, the docker file installs Chromium without requiring a specific version.

I was recently made aware of the fact that some commercial software products that rely on Chromium for HTML->PDF conversion are now failing due to Chromium randomly getting stuck since a recent Chromium update.

I have no issues with Gotenberg, but out of curiosity I checked the build Dockerfile and if I am not mistaken it installs whatever is the current Chromium version. This could be potentially risky.

Also, images are supposed to be immutable, is Chromium auto-update disabled? Probably yes because I guess the image is based on Debian and Debian uses APT for updating Chromium rather than letting Chromium update itself, correct?

Thank you.

**Comment by gulien** [comment, marked as answer]
Hello @giamma 👋

According to the [official documentation](https://www.chromium.org/administrators/turning-off-auto-updates/):

> Turning off Auto-Updates on Linux
>
> Google Chrome and Chromium are not auto-updated automatically on Linux; your package manager handles this.

That being said, a new version of Gotenberg may bring a new version of Google Chrome / Chromium.

**Reply by giamma** [reply]
Hi @gulien thank you for answer. 

I am not worried about new version of gotenberg, I am sure you run your tests before releasing and I do the same for my product when adopting a new version of gotenberg. 

I just wanted to doublecheck that Chrome will not auto update :-)
Thank you.
